ng-dev/release/publish/external-commands.ts

```diff
@@ -36,8 +36,15 @@ import {FatalReleaseActionError} from './actions-error';
 /**
  * Invokes the `ng-dev release set-dist-tag` command in order to set the specified
  * NPM dist tag for all packages in the checked out branch to the given version.
+ *
+ * Optionally, the NPM dist tag update can be skipped for experimental packages. This
+ * is useful when tagging long-term-support packages within NPM.
  */
-export async function invokeSetNpmDistCommand(npmDistTag: NpmDistTag, version: semver.SemVer) {
+export async function invokeSetNpmDistCommand(
+  npmDistTag: NpmDistTag,
+  version: semver.SemVer,
+  options: {skipExperimentalPackages: boolean} = {skipExperimentalPackages: false},
```

Comment:
So for setting the npm tag we are set up to skip experimental packages for LTS, but when we cut an LTS version we still publish the package tagged as LTS.

If `@angular/package` is experimental, we publish `0.100.0`, then we publish `0.200.0` but `0.100.0` isn't tagged as `lts-v1`.  Then when we release `0.100.1` as LTS it would be published with `lts-v1` tag.

We should add this same logic to not publish experimental in `CutLongTermSupportPatchAction` as well.

Reply:
I think for the sake of simplicity and keeping version numbers in sync, we should still publish such experimental packages in those cases. It should not have any negative impact, but to simplify our setup.

We are just omitting the LTS NPM dist tags so that we do not indicate any LTS guarantees / have people install the package through e.g @12-lts.

Reply:
In `Action._publishBuiltPackageToNpm` we have:
```ts
await runNpmPublish(pkg.outputPath, npmDistTag, this.config.publishRegistry);
```

So when we publish an LTS (after the initial retag during the major release), we would begin to tag it as `{version}-lts`.  So I believe that we need to add this same skipping logic into the npm publishing process.  I might be misunderstanding what we are looking to avoid.

Reply:
ah I see! good point. 

yeah, that is something we need to address. Maybe if we even go that far to exclude experimental NPM packages in such cases, should we also consider what I thought you proposed initially? 

i.e. we do not actually release the experimental package for an LTS? that would mean the package is not in sync with the other packages, but I'm trying to understand if that is acceptable/desirable? any thoughts?

Reply:
updated